backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -30,6 +30,8 @@ Restoring to a previous cluster state is a destructive and destablizing action t
 
 Prior to performing a restore, see xref:../../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-scenario-2-restoring-cluster-state-about_dr-restoring-cluster-state[About restoring cluster state] for more information on the impact to the cluster.
 ====
+
```

Comment:
I would suggest putting this above the `Restoring to previous state` section since it's stated as being the last resort